pandas-stubs/core/arrays/categorical.pyi

```diff
@@ -168,7 +168,6 @@ class Categorical(ExtensionArray, PandasObject):
     def isnull(self) -> np_ndarray_bool: ...
     def notna(self) -> np_ndarray_bool: ...
     def notnull(self) -> np_ndarray_bool: ...
-    def put(self, *args, **kwargs) -> None: ...
```

Comment:
I'm guessing that a lot more methods in `class Categorical` could be deleted as well, but maybe that's hard to determine